dev_tools/autogenerate-bloqs-notebooks-v2.py

```diff
@@ -339,6 +340,21 @@
         bloq_specs=[qualtran.bloqs.factoring.mod_mul._MODMUL_DOC],
         directory=f'{SOURCE_DIR}/bloqs/factoring',
     ),
+    NotebookSpecV2(
+        title='Elliptic Curve Cryptography',
+        module=qualtran.bloqs.factoring.ecc,
+        bloq_specs=[
+            qualtran.bloqs.factoring.ecc._find_ecc_private_key._ECC_BLOQ_DOC,
+            qualtran.bloqs.factoring.ecc._ec_phase_estimate_r._EC_PE_BLOQ_DOC,
+            qualtran.bloqs.factoring.ecc._ec_add_r._ECC_ADD_R_BLOQ_DOC,
+            qualtran.bloqs.factoring.ecc._ec_add_r.EC_WINDOW_ADD_BLOQ_DOC,
```

Comment:
are you missing the leading underscore here?

Reply:
ah good catch